### PR TITLE
bug(zenko-699): Fix Lifecycle Test

### PR DESCRIPTION
**Pull request by tmacro**

### CODE REVIEW

File: tests/local-values.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+nodeCount: &nodeCount 1
```
Comment (ssalaues) [inline tests/local-values.yml]: Is this for local testing?

File: tests/local-values.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    name: default
+  replicas:
+    servers: *nodeCount
+    sentinels: *nodeCount
+
+cloudserver-front:
+  image:
+    pullPolicy: Always
+  orbit:
+    endpoint: "http://ciutil-orbit-simulator:4222"
```
Comment (ssalaues) [inline tests/local-values.yml]: If this is for local testing do we want orbit-simulator references?

File: tests/local-values.yml (hunk 1 above)
Comment (tmacro) [reply tests/local-values.yml]: Yes

File: tests/local-values.yml (hunk 2 above)
Comment (tmacro) [reply tests/local-values.yml]: orbit sim still runs for local test runs

File: tests/local-values.yml (hunk 1 above)
Comment (ssalaues) [reply tests/local-values.yml]: Just posting here the discussion Taylor and I had offline:
The concern was fundamentally changing an existing cluster when running this make command but it seems like it already previously had this intended affect. It's out of the scope of this PR but we should look into changing up the Makefile to better suit our needs.

**Pull request merged**